scripts/fetch_snet_waveform.py

```
@@ -80,7 +80,9 @@
 SEGMENTS_RECENT = 4            # 4 segments for recent days (every 6h)
 SEGMENTS_BACKFILL = 1          # 1 segment for backfill days (daily average)
 RECENT_DAYS = 7                # Last 7 days get high-resolution sampling
-MAX_BACKFILL_DAYS_PER_RUN = 5  # 5 days × 3 codes = 15 backfill requests (reduced to fit 6h job limit)
+MAX_BACKFILL_DAYS_PER_RUN = int(os.environ.get("SNET_MAX_BACKFILL_DAYS", "20"))
+# Worst case: 20 backfill × 3 sensors + 7 recent × 4 segments × 3 sensors = 144
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)